.github/workflows/call-report.yml

```diff
@@ -0,0 +1,72 @@
+name: "Test calling report.yml"
```

Comment:
Can you add a comment above this which briefly summarizes what this workflow is doing / used for, similar to the other files?